last_seen.py

```diff
@@ -0,0 +1,32 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+from time import strftime, gmtime
+
+
+class LastSeen(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.seen = self.open_storage('last_seen')
+
+    @event
+    def seen_evt(self, event):
+        events = ['quit', 'part', 'kick', 'join', 'rename', 'topic', 'message']
+        if event.event_type in events:
+            nick = event.meta['nick']
+            time = gmtime()
+            evt_type = event.event_type
+            self.seen[nick] = {time: evt_type}
```

Comment:
@jakubn I think a basic touple `(time, evt_type)` would be enough. You could then access  it via `self.seen[nick][0]` for instance. The 1st index would return `event_time`.